python_scripts/01_tabular_data_exploration.py

```diff
@@ -283,31 +295,34 @@ def plot_tree_decision_function(tree, X, y, ax=None):
 plot_tree_decision_function(tree, X, y)
 
 # %% [markdown]
-# Allowing only 3 leaves in the tree, we get similar rules to the ones we
+# By allowing only 3 leaves in the tree, we get similar rules to the ones we
 # designed by hand:
-# * the persons younger than 28.5 year-old will be considered in the class
-#   earning `<= 50K`.
-# * the persons older than 28.5 and working more than 40.5 hours-per-week
-#   will be considered in the class earning `> 50K`, while the persons working
-#   below 40.5 hours-per-week, will be considered in the class
+# * the persons younger than 28.5 year-old (X[0] < 28.5) will be considered in the class
 #   earning `<= 50K`.
+# * the persons older than 28.5 and working less than 40.5 hours-per-week (X[1] <= 40.5)
+#   will be considered in the class earning `<= 50K`, while the persons working
+#   above 40.5 hours-per-week, will be considered in the class
+#   earning `> 50K`.
 
 # %% [markdown]
 #
 # In this notebook we have:
 # * loaded the data from a CSV file using `pandas`
-# * looked at the kind of variables in the dataset, and differentiated
+# * looked at the differents kind of variables to differentiate
 #   between categorical and numerical variables
 # * inspected the data with `pandas`, `seaborn` and `pandas_profiling`. Data inspection
 #   can allow you to decide whether using machine learning is appropriate for
 #   your data and to highlight potential peculiarities in your data
 #
 # Key ideas discussed:
+# * A first glance to the data can give some insights of features importance
+# * It is possible to visualize the decision rules from a decision tree
```

Comment:
In line with my previous comment, I would generalize the idea here: we saw that a machine learning model can come with rules similar to a human would have done.

Reply:
I remove this part